azurerm/data_source_policy_definition.go

```diff
@@ -0,0 +1,80 @@
+package azurerm
+
+import (
+	"fmt"
+
+	"github.com/Azure/azure-sdk-for-go/services/resources/mgmt/2018-05-01/policy"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func dataSourceArmPolicyDefinition() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceArmPolicyDefinitionRead,
+		Schema: map[string]*schema.Schema{
+			"display_name": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"management_group_id": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+			"name": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+			"type": {
+				Type:     schema.TypeString,
+				Optional: true,
```

Comment:
As this is a data source this should be computed not optional
```suggestion
				Computed: true,
```